.docker/Dockerfile.govcms8

```diff
@@ -15,7 +15,9 @@ RUN sed -i -e "/govcms\/govcms/ s!^1.0!${GOVCMS_PROJECT_VERSION}!" /app/composer
 COPY scripts/composer/ScriptHandler.php /app/scripts/composer/ScriptHandler.php
 
 ENV COMPOSER_MEMORY_LIMIT=-1
-RUN composer install -d /app && composer clearcache
+RUN composer install -d /app \
```

Comment:
Seems logical and not too weird. But can we have a comment in the code about why we do this please.

Reply:
done!